engine/pkg/dbutil/config.go

```diff
@@ -33,8 +33,8 @@ type DBConfig struct {
 }
 
 // DefaultDBConfig creates a default DBConfig
-func DefaultDBConfig() DBConfig {
-	return DBConfig{
+func DefaultDBConfig() *DBConfig {
```

Comment:
Why do we need pointer here?

Reply:
Both is ok. Only for less copy.

Reply:
For small structures, it may be safer and more efficient to use value assignment.